603-spring-web-smallrye-openapi/src/main/java/io/quarkus/qe/books/persistence/model/Book.java

```diff
@@ -55,35 +57,40 @@ public void setAuthor(String author) {
 
     @Override
     public int hashCode() {
-        final int prime = 31;
-        int result = 1;
-        result = prime * result + ((author == null) ? 0 : author.hashCode());
-        result = prime * result + (int) (id ^ (id >>> 32));
-        result = prime * result + ((title == null) ? 0 : title.hashCode());
-        return result;
+        return Objects.hash(author, id, title);
     }
 
     @Override
     public boolean equals(Object obj) {
```

Comment:
I know it is beyond the scope of this PR, but I would consider to use some 3rd-party help here:
```
        org.apache.commons.lang3.builder.EqualsBuilder.reflectionEquals(this, obj);
```
WDYT?

Reply:
I also think the equals methods should be refactored... but as you said, not of scope of this PR. Feel free to create a card in Trello and propose it, or maybe also something to discuss in our next retrospective.